backend/kernelCI_app/management/commands/update_db.py

```diff
@@ -46,7 +44,7 @@ def __init__(self, *args, **kwargs):
         self.end_interval = None
         self.start_timestamp = None
         self.end_timestamp = None
-        if is_boolean_or_string_true(USE_DASHBOARD_DB):
+        if USE_DASHBOARD_DB:
```

Comment:
Oh, please fix the import. Settings shouldn't be imported as a module, otherwise they will break the `--settings` CLI argument and `override_settings` in unittests

```diff
-from kernelCI.settings import USE_DASHBOARD_DB
+from django.conf import settings
```

```suggestion
        if settings.USE_DASHBOARD_DB:
```